flow/project.py

```diff
@@ -2498,6 +2499,8 @@ def submit_operations(self, operations, _id=None, env=None, parallel=False, flag
             _id = self._store_bundled(operations)
         if env is None:
             env = self._environment
+        else:
+            print("DeprecationWarning: Providing the env argument to submit and submit_operations is deprecated. Set the environment while constructing a FlowProject", file=sys.stderr)
```

Comment:
Please use a `DeprecationWarning` class instead of just a message that says "DeprecationWarning." For an example, see below. I based this message text on the format used by the `deprecation` package. We use that package for deprecating methods, but it is not possible to use that package for deprecating a single argument.
```suggestion
            warnings.warn("The env argument is deprecated as of 0.10 and will be removed in 0.12. "
                          "Instead, set the environment when constructing a FlowProject.",
                          warnings.DeprecationWarning)
```